Database

phambaokien1997 opened the pull request:
Not Yet

phambaokien1997 committed:
Database

Not Yet

NightCatcher commented on Core/BookStore.Core/Database/Book.cs:
```diff
@@ -1,18 +1,34 @@
-        [ForeignKey("AuthorId")]
-        public virtual Author Author { get; set; }
-    }
+		public decimal Price { get; set; }
```
format code (ctrl K + D)

NightCatcher commented:
try to add migrations files for these entity classes using the command in the read me section

phambaokien1997 committed:
Merge branch 'master' into NewDatabase

phambaokien1997 committed:
add migration for related table and entity

phambaokien1997 committed:
ngáo đá đám chết cmm giờ vcc

rứa cái lz ni là chi ? mô tả hà? mô tả ni là hắn hiện ra ở mô trong git ?

tại răng thay vì hỏi thì mình ko thử phát hè

phambaokien1997 committed:
Update Data And Migration
